source/IL2CPU.API/LabelName.cs

```diff
@@ -240,7 +256,7 @@ public static string GetFullName(MethodBase aMethod)
 
         public static string GetFullName(FieldInfo aField)
```

Comment:
should this method also have bool aAssemblyIncluded = true to make it more flexible or will it always be false here

Reply:
It will always be true in this case, because we want assembly name to be included when generating code so that labels are unique but not when generating functions. The assumption is that functions with the same namespace etc are actually just the same function functionality wise (so we can use the same plugs etc).